SPECS-EXTENDED/flac/flac.spec

```diff
@@ -1,33 +1,27 @@
+Summary:        An encoder/decoder for the Free Lossless Audio Codec
+Name:           flac
+Version:        1.4.3
+Release:        1%{?dist}
+License:        BSD-3-Clause AND GPL-2.0-or-later AND GFDL-1.1-or-later
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-# Disable if you don't need xmms
-%global with_xmms 0
-
-%if %{with_xmms}
-%define xmms_inputdir %(xmms-config --input-plugin-dir 2>/dev/null || echo %{_libdir}/xmms/General)
-%endif
-
-Summary: An encoder/decoder for the Free Lossless Audio Codec
-Name: flac
-Version: 1.3.4
-Release: 1%{?dist}
-License: BSD and GPLv2+ and GFDL
-Source0: https://downloads.xiph.org/releases/flac/flac-%{version}.tar.xz
-URL: https://www.xiph.org/flac/
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}
-BuildRequires: libogg-devel
-BuildRequires: gcc gcc-c++ automake autoconf libtool gettext-devel doxygen
-%if %{with_xmms}
-BuildRequires: xmms-devel desktop-file-utils
-Source1: xmms-flac.desktop
-%endif
+URL:            https://www.xiph.org/flac/
+Source0:        https://downloads.xiph.org/releases/flac/flac-%{version}.tar.xz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  doxygen
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  gettext-devel
+BuildRequires:  libogg-devel
+BuildRequires:  libtool
+Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
 %ifarch %{ix86}
```

Comment:
I assume this is for 32 bit?  Since we touched this line, we can probably remove it?

Reply:
Yes; sure .. might as well :)